tests/ethpm/test_package.py

```diff
@@ -21,16 +21,19 @@ def deployed_safe_math(safe_math_package, w3):
 
 
 def test_package_object_instantiates_with_a_web3_object(all_manifests, w3):
-    current_package = Package(all_manifests, w3)
-    assert current_package.w3 is w3
+    package = Package(all_manifests, w3)
+    assert package.w3 is w3
 
 
-def test_set_default_web3(all_manifests, w3):
+def test_set_default_web3(deployed_safe_math, w3):
     new_w3 = Web3(Web3.EthereumTesterProvider())
-    current_package = Package(all_manifests, w3)
-    assert current_package.w3 is w3
-    current_package.set_default_w3(new_w3)
-    assert current_package.w3 is new_w3
+    original_package, _ = deployed_safe_math
+    assert original_package.w3 is w3
+    new_package = original_package.set_default_w3(new_w3)
+    assert new_package.w3 is new_w3
+    assert original_package.manifest == new_package.manifest
```

Comment:
You can do one more check of `assert new_package is not original_package` since they should be distinct objects.